.circleci/config.yml

```diff
@@ -1,9 +1,75 @@
+orbs:
+  # The Windows orb gives us everything we need to start using the Windows executor.
+  win: circleci/windows@2.4.0
+
 defaults: &defaults
   docker:
     - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:go1.16-tf1.0-tg0.30
 
-version: 2
+version: 2.1
 jobs:
+  test_windows:
+    executor:
+      name: win/default
+      size: "large"
+    steps:
+      - checkout
+      - run:
+          name: Install golang
+          shell: powershell.exe
+          command: |
+            # Remove the old go installation
```

Comment:
Could you extract these scripts into a separate file or files? I don't think we need to make them generic and put them in `terraform-aws-ci` _yet_, but having them in separate files will help in readability, maintainability, and set us on the path to making things more generic.

Reply:
Absolutely.

Reply:
Done initially in 16a288a